api/HTMLInputElement.json

```diff
@@ -643,7 +643,7 @@
               "version_added": "56"
             },
             "firefox_android": {
-              "version_added": true
+              "version_added": "56"
```

Comment:
Looks like all of the `labels` entries (on multiple elements) were "56" for both Firefoxes, and this was the only outlier. Good!